inc/fluent_bit/class-msgpackformatter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Altis Cloud Fluent Bit MsgPack formatter.
+ *
+ * @package altis/cloud
+ */
+
+namespace Altis\Cloud\Fluent_Bit;
+
+/*
+ * TODO Copies some formatter from Monolog, need to give credit
+ * This file is part of the Monolog package.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Monolog\Formatter\FluentdFormatter;
+use RuntimeException;
+
+/**
+ * Class MsgPackFormatter
+ *
+ * Serializes a log message to Fluent Bit socket protocol
+ *
+ * @author Nathaniel Schweinberg <nathaniel@humanmade.com>
+ */
+class MsgPackFormatter extends FluentdFormatter {
+
+	/**
+	 * Construct the MsgPackFormatter class.
+	 *
+	 * @param boolean $levelTag whether to append log level to Fluent Bit tag
+	 * name or not.
+	 * @return self
+	 * @throws RuntimeException Break when msgpack extension is not available
+	 */
+	public function __construct( bool $levelTag = false ) { // phpcs:ignore WordPress.NamingConventions.ValidVariableName
+		if ( ! function_exists( 'msgpack_pack' ) ) {
+			throw new RuntimeException( 'PHP\'s msgpack extension is required to use Monolog\'s MsgPackFormatter' );
+		}
+
+		// phpcs:ignore WordPress.NamingConventions.ValidVariableName
+		$this->levelTag = $levelTag;
+	}
```

Comment:
Doesn't seem like there's a good reason to ignore the coding standard for the parameter name. This _might_ now throw another error for the property name, but given that's inherited, that specific part can be ignored (I think it's a NamingConventions.ValidPropertyUse or something)

```suggestion
	 * @param boolean $level_tag whether to append log level to Fluent Bit tag
	 * name or not.
	 * @return self
	 * @throws RuntimeException Break when msgpack extension is not available
	 */
	public function __construct( bool $level_tag = false ) {
		if ( ! function_exists( 'msgpack_pack' ) ) {
			throw new RuntimeException( 'PHP\'s msgpack extension is required to use Monolog\'s MsgPackFormatter' );
		}

		$this->levelTag = $level_tag;
	}
```